notebooks/book2/02/zipfs_law_plot.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    N = jnp.sum(wordcounts)
</pre><p>what is c, x, N, a here? please use meaning full variable names</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
'**a**' and '**x**' are the variables as shown in the book for the Zipf law equation so could you please suggest to me a different name for them. Also, c is the rank from which the Zipf law plotting starts and  N is the total number of words in the passage so is the name: "**start_rank**" and " **total_words**" fine?

Reply:
Yes, you can put the variable names by your own.